recipes/alps/meta.yaml

```diff
@@ -0,0 +1,76 @@
+{% set name = "alps" %}
+{% set version = "2.3.b1" %}
+{% set revision = "r7782" %}
+{% set sha256 = "69910b4fcd5e98a01741843de530f896864a390f693bf010b2db56ceb3f071d7" %}
+{% set mpi = os.environ.get('MPI') or 'mpich' %}
+{% set blas = os.environ.get('BLAS') or 'openblas' %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}-{{revision}}-src.tar.gz
+  url: http://alps.comp-phys.org/static/software/releases/{{ name }}-{{ version }}-{{revision}}-src.tar.gz
+  sha256: {{ sha256 }}
+  patches:  # [win]
+    - no_install_deplibs.patch  # [win]
+    - boost_defs.patch  # [win]
+    - hdf5_win.patch  # [win]
+    - mps_win.patch  # [win]
+    - r7818.patch  # [win]
+    - cstdint_types.patch  # [win]
+    - parapack_win.patch  # [win]
+
+build:
+  skip: true  # [not py27]
+  skip: true  # [not win]
+  number: 0
+  detect_binary_files_with_prefix: True
+  features:
+    - blas_{{blas}}  # [not win]
+
+
+requirements:
+  build:
+    - toolchain
+    - cmake
+    - boost
+    - python
+    - numpy
+    - scipy
+    - matplotlib
+    - hdf5
+    - blas 1.* {{blas}}  # [not win]
+  run:
+    - boost
+    - python
+    - numpy
+    - scipy
+    - matplotlib
+    - hdf5
+    - blas 1.* {{blas}}  # [not win]
```

Comment:
Certain dependencies that you use (eg HDF5) require certain versions to be used so that they are consistent across all of conda-forge. You can find the current versions here https://github.com/conda-forge/conda-forge.github.io/blob/master/scripts/pin_the_slow_way.py#L31